yaml/xpu_functions.yaml

```diff
@@ -113,6 +113,7 @@ supported:
   - gelu.out
   - gelu_backward
   - gelu_backward.grad_input
+  - silu
```

Comment:
Do you plan to add variants? For example, huggingface examples uses `silu.out`. See https://github.com/pytorch/pytorch/issues/127941.

Reply:
I am confused. There is #389 already adding `silu` and variants? Why do you open another PR? Please, close one of them or add a reason to have both, for example, one might be a draft or experimental.

Reply:
It‘s an example PR for showing how we make integration of SYCL in PyTorch. Please refer https://github.com/intel/torch-xpu-ops/pull/389